src/ploomber_engine/ipython.py

```diff
@@ -146,6 +164,19 @@ class PloomberClient:
     display_output : bool, default=False
         If True, it prints the same output the notebook prints.
 
+    progress_bar : bool, default=True
+        Display a progress bar.
+
+    debug_later : bool or str, default=False
```

Comment:
I think there can be confusion if `debug_later` can be a bool or a path

Reply:
i thought about it, but adding two parameters (one bool, and one path) seems too much. thoughts?

Reply:
I see your point.
I think if someone needs to store the output in a different path/file, he wouldn't mind writing an "extra" argument for specifying the path (maybe '--save' ?)

Reply:
I think let's give it a try. This API is not the primary one (only for advanced users). and it'll be easier to migrate them to a new one that has the extra parameter than the other way around.